### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by GkvJeep**

### CODE REVIEW

File: sdrgui/CMakeLists.txt [hunk 1]
```diff
@@ -49,7 +49,7 @@ set(sdrgui_SOURCES
     gui/valuedialz.cpp
 
     dsp/scopevis.cpp
-#   dsp/scopevismulti.cpp
+    dsp/scopevismulti.cpp
```
Comment (f4exb) [inline sdrgui/CMakeLists.txt]: Why? This is of no use at this moment.

File: plugins/samplesource/CMakeLists.txt [hunk 2]
```diff
@@ -78,7 +78,7 @@ else(LIBUSB_FOUND AND LIBMIRISDR_FOUND)
 endif(LIBUSB_FOUND AND LIBMIRISDR_FOUND)
 
 find_package(SoapySDR)
-if(LIBUSB_FOUND AND SOAPYSDR_FOUND)
+if(SOAPYSDR_FOUND)
```
Comment (f4exb) [inline plugins/samplesource/CMakeLists.txt]: Fine but if this is the only change I'd rather patch this file than do a merge

File: sdrgui/CMakeLists.txt (hunk 1 above)
Comment (GkvJeep) [reply sdrgui/CMakeLists.txt]: Error compile msvc.

File: plugins/samplesource/CMakeLists.txt (hunk 2 above)
Comment (GkvJeep) [reply plugins/samplesource/CMakeLists.txt]: Many devices when using Soapysdr do not require libusb.
Libusb preinstalled in Soapyusdr.

File: sdrgui/CMakeLists.txt (hunk 1 above)
Comment (GkvJeep) [reply sdrgui/CMakeLists.txt]: Error at the stage of linker, there is a link from other files.

File: plugins/samplesource/CMakeLists.txt (hunk 2 above)
Comment (GkvJeep) [reply plugins/samplesource/CMakeLists.txt]: The same is required for  plugins\samplesink\CMakeLists.txt
line 32   if(LIBUSB_FOUND AND SOAPYSDR_FOUND)

**Comment by f4exb** [comment]
All changes implemented manually in v4.3.2

**Pull request closed without merging**